holmes/config.py

```diff
@@ -193,6 +201,7 @@ def _load_default_robusta_config(self):
         if self._should_load_robusta_ai() and self.api_key:
```

Comment:
we need also `self._load_default_robusta_config = ROBUSTA_AI_MODEL_NAME` or something

Reply:
Not sure i understand can you please explain?